apps/nextjs-website/react-components/components/Form/Form.tsx

```diff
@@ -249,156 +250,170 @@ const Form = ({
   return (
     <Box
       sx={{
-        maxWidth: 900,
-        mx: 'auto',
-        p: 4,
-        textAlign: 'center',
-        borderRadius: 2,
-        boxShadow: 3,
-        color: textColor,
+        width: '100vw',
         position: 'relative',
+        left: '50%',
+        right: '50%',
+        marginLeft: '-50vw',
+        marginRight: '-50vw',
```

Comment:
```suggestion
```
I don't believe these do anything.

left and right only work with absolute or fixed positioning.
marginLeft and marginRight, as set, would effectively make the section have 0 width, without actually moving it. So they end up being useless (if not dangerous).